MovieApp/MovieApp/App/Features/MovieBrowser/Components/MovieSearchBar+Design.swift

```diff
@@ -0,0 +1,139 @@
+import UIKit
+import SnapKit
+
+extension MovieSearchBar: DesignProtocol {
+    
+    func buildViews() {
+        createViews()
+        styleViews()
+        defineLayoutForViews()
+    }
+    
+    func createViews() {
+        searchGrayFieldView = UIView()
+        addSubview(searchGrayFieldView)
+        
+        let image = ImageEnum.searchBarIcon.image
+        searchBarIconImageView = UIImageView(image: image)
+        searchGrayFieldView.addSubview(searchBarIconImageView)
+        
+        searchTextField = UITextField()
+        searchTextField.delegate = self
+        searchGrayFieldView.addSubview(searchTextField)
+        
+        cancelButton = UIButton()
+        cancelButton.addTarget(self, action: #selector(cancelPressed), for: .touchUpInside)
+        addSubview(cancelButton)
+        
+        deleteTextButton = UIButton()
+        deleteTextButton.addTarget(self, action: #selector(deleteTextFieldEntry), for: .touchUpInside)
+        searchGrayFieldView.addSubview(deleteTextButton)
+    }
+    
+    func styleViews() {
+        searchGrayFieldView.layer.cornerRadius = grayFieldCornerRadius
+        searchGrayFieldView.backgroundColor = .searchFieldGray
+        
+        searchBarIconImageView.contentMode = .scaleAspectFit
+        
+        searchTextField.attributedPlaceholder = NSAttributedString(
+            string: "Search",
+            attributes: [NSAttributedString.Key.foregroundColor: UIColor.searchPlaceholderColor,
```

Comment:
You should be able to write attributes keys without `NSAttributedString.Key` just `.foregroundColor`.

Reply:
Changed [here](https://github.com/lihter/Movie-App/pull/24/files#diff-6a7e0c3cc088b3a980b8a92c08ad5d190f909d67f14aa13f150bdc992ab52302R40) and on every other occurrence in code.